java/core/src/java/org/apache/orc/impl/RunLengthIntegerWriterV2.java

```diff
@@ -530,7 +531,7 @@ private void determineEncoding() {
       // fallback to DIRECT encoding.
       // The decision to use patched base was based on zigzag values, but the
       // actual patching is done on base reduced literals.
-      if ((brBits100p - brBits95p) != 0 && Math.abs(min) < BASE_VALUE_LIMIT) {
+      if ((brBits100p - brBits95p) != 0) {
```

Comment:
Thanks. Look like this is the correct way to resolve the problem instead of this workaround. w